feat(issue-priority): Add new page for issue priority (#9452)
getsentry/sentry-docs: docs/product/issues/issue-priority/index.mdx

---
title: Issue Priority
sidebar_order: 31

> description: "Learn about issue priority."

Reviewer: ```suggestion
description: "Learn how Sentry determines the level of issue severity and prioritizes issue importance accordingly."
```

---

<Note>

This feature is only available to [Early Adopters](/product/accounts/early-adopter-features/) at this time.

</Note>

Issue priority helps you identify and address critical errors impacting your application's functionality and user experience. By assigning a priority level to each issue, Sentry can surface the most pressing matters first, allowing your development team to focus on resolving impactful problems efficiently.

## How Priority Works

> 
> There are three discrete priority levels for issues in Sentry:
> 
> - **High**: Issues that are likely to be actionable and require immediate attention.
> - **Medium**: Issues that are likely to be actionable and require attention in the near future.
> - **Low**: Issues that do not require immediate attention.
> 
> ### Priority Assignment on Issue Creation
> 
> When Sentry receives an event, it assigns a priority to the issue based on the event's log level (for error issues) or severity (non-error issues like performance). If the event's severity is high or the log level is `error` or `fatal`, the issue is assigned a high priority. If the severity is medium or the log level is `warning`, the issue is assigned a medium priority. If the severity is low or the log level is `info` or `debug`, the issue is assigned a low priority.

Reviewer: ```suggestion

When Sentry receives an event, it assigns a priority to the issue based on the event's **log level** (for error issues) or **severity** (for non-error issues like performance). There are three discrete priority levels for issues in Sentry:

- **High**: Issues that are likely to be actionable and require immediate attention. Issues are determined to be high priority if the event's severity is high or the log level is `error` or `fatal`.
- **Medium**: Issues that are likely to be actionable and require attention in the near future. Issues are determined to be medium priority if the severity is medium or the log level is `warning`.
- **Low**: Issues that don't require immediate attention. Issues are determined to be low priority if the severity is low or the log level is `info` or `debug`.
```

> ### Automatic Priority Updates
> 
> Sentry continuously monitors the volume of events for each issue. If an issue experiences a surge in occurrences (escalates), its priority will be automatically bumped to reflect its growing significance. See the [escalating issues algorithm](/product/issues/states-triage/escalating-issues/) for more details on how escalations are identified.
> 
> When an issue is no longer escalating, its priority will revert to the previous level.
> 

Reviewer: ```suggestion
### Automatic Priority Updates

Sentry continuously monitors the volume of events for each issue. If there's a surge of events for a particular issue (it escalates), its priority level will be automatically bumped up. See the [escalating issues algorithm](/product/issues/states-triage/escalating-issues/) for more information on how escalations are identified. When an issue de-escalates, its priority will go back down to the previous level.

```

### Manual Priority Updates

> 
> You can manually update the priority of an issue at any time, either from the issue details page or from the issue stream. Once an issue's priority has been manually updated, it will no longer be automatically adjusted based on event volume.

Reviewer: ```suggestion

You can update issue priority manually at any time, either from the issue details page or from the issue stream. But once an issue's priority has been changed manually, it will no longer be automatically adjusted if the event volume escalates.
```

> ## Priority in the Issue Stream
> 
> By default, the issue stream filters out low-priority issues. If you would like to see low-priority issues by default, you can [set your default search](/product/reference/search/saved-searches/#set-a-default-search) to include them.
> 
> When viewing the "Prioritized" tab, you can quickly triage issues by using the priority dropdown. You may also update priority of multiple issues at once by selecting them and using the bulk action dropdown. Low-priority issues will disappear from the "Prioritized" tab once their priority is updated.
> 

Reviewer: ```suggestion
## Priority in the Issue Stream

By default, the issue stream on the **Issues** page filters out low-priority issues. If you'd like to be able to see low-priority issues, you can [set your default search](/product/reference/search/saved-searches/#set-a-default-search) to include them.

You can quickly filter the type of issue you'd like to view using the priority dropdown in the "Prioritized" tab on the **Issues** page. You may also update the priority level of multiple issues at once by selecting them and using the bulk action dropdown. Low-priority issues will disappear from the "Prioritized" tab once their priority is updated.

```
It might be good to show the dropdown image in the context of the page as a whole so people don't think it's a top nav tab.

<img width="1004" alt="Screenshot 2024-03-18 at 4 21 09 PM" src="https://github.com/getsentry/sentry-docs/assets/26800704/e55ea46e-97ad-44ec-9575-485420319504">

Also, might be good to show an image of multiple issues selected and the bulk action dropdown.
